tools/testbuild.sh

```diff
@@ -218,6 +236,32 @@ function makefunc {
   return $fail
 }
 
+function checkfunc {
+  build_cmd="${MAKE} ${MAKE_FLAGS} \"${EXTRA_FLAGS}\" ${JOPTION} 1>/dev/null"
+
+  local config_sub_path=$(echo "$config" | sed "s/:/\//")
+  local sub_target_name=${config_sub_path#$(dirname "${config_sub_path}")/}
+  local codechecker_logs_dir=${ARTIFACTDIR}/codechecker_logs/${config_sub_path}
```

Comment:
```suggestion
  local codechecker_dir=${ARTIFACTDIR}/codechecker
```

Reply:
why need config_sub_path? let's remove it.

Reply:
If there are multiple targets in codechecker.dat, the results of the checking may make the folder look disordered.
Now it's a separate folder for each target.

Reply:
Ok, let's change codechecker_logs_dir to codechecker_dir

Reply:
that's ok